watchgod/watcher.py

```diff
@@ -41,6 +42,9 @@ def _watch_file(self, path, changes, new_files):
             changes.add((Change.modified, path))
 
     def _walk_dir(self, dir_path, changes, new_files):
+        if dir_path in self.ignored_paths:
```

Comment:
`ignore_paths` should be a set, not a list, containment checks are massively faster for sets than lists.